be/src/common/config.h

```diff
@@ -913,6 +913,10 @@ CONF_Int32(max_depth_in_bkd_tree, "32");
 CONF_Int32(num_broadcast_buffer, "32");
 // semi-structure configs
 CONF_Bool(enable_parse_multi_dimession_array, "true");
+
+// Report a tablet as bad when io errors occurs more than this value.
+CONF_mInt64(max_tablet_io_errors, "-1");
```

Comment:
More comments are needed to explain the function of this switch, including:
1 When this case appears ENV Prod, it is better for the administrator to deal it.
2 This switch is off by default, before turning on the switch, make sure you can understand the risks caused by this switch, this risk is : this switch may catch the exception and  digest it, this may cause some other case.